recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "7.0.7" %}
-{% set lab_version = ">=4.0.7,<5" %}
+{% set version = "7.0.8" %}
+{% set lab_version = ">=4.0.7,<4.1" %}
```

Comment:
Relevant upstream PR: https://github.com/jupyter/notebook/pull/7249

Reply:
prolly needs a `repodata-patches` PR, otherwise solver will just pull older ones.

Reply:
(but this should land here first)

Reply:
ready to go, but still in draft: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/662